content/extending-chat-widget/custom-identity-provider/index.mdx

```diff
@@ -0,0 +1,174 @@
+---
+weight: 60
+category: "extending-chat-widget"
+subcategory: "custom-identity-provider"
+title: "Custom Identity Provider"
+tagline: "Pair your own customer identity with LiveChat"
+desc: "Cross-platform chats for your applications"
+versionGroup: "chat-widget"
+---
+
+# Introduction
+
+## What is it for?
+
+If you have a more advanced service, handling user sessions, you can pair LiveChat customer identity with your own. As a result, your customers will never lose chats they made with your support. What is more, if you also use LiveChat on other platfroms (e.g., in mobile applications), their conversations will sync, making the communication even more seamless and natural.
```

Comment:
I would add information that this kind of integration requires running its own backend service and it requires some development skills. As an alternative, we should mention the possibility to pass customer email/name using JS API.

Reply:
What is the relation between identtiy provider and customer email actually? One handles authorization - pass of identity created by our backend the other is just metadata that can be filled by customer